package.json

```diff
@@ -4,15 +4,16 @@
   "description": "",
   "main": "index.js",
```

Comment:
`index.js` is being deleted. Is this line still necessary?

I'm still learning npm; this is definitely a question, not a correction.

Reply:
I thought it might be required, but I [double-checked](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#main) and it is not. Removed